tests/lib/lte_lc/src/main.c

```diff
@@ -375,14 +375,16 @@ static void test_parse_ncellmeas(void)
 {
 	int err;
 	char *resp1 = "%NCELLMEAS: 0,\"021D140C\",\"24201\",\"0821\",65535,5300,"
-			"449,50,15,10891,5300,194,46,8,0,1650,292,60,27,24";
-	char *resp2 = "%NCELLMEAS: 1,\"021D140C\",\"24201\",\"0821\",65535,5300";
-	char *resp3 = "%NCELLMEAS: 0,\"071D340C\",\"24201\",\"0821\",65535,5300,449,50,15,10891";
+			"449,50,15,10891,5300,194,46,8,0,1650,292,60,27,24,10000";
+	char *resp2 = "%NCELLMEAS: 1,\"021D140C\",\"24201\",\"0821\",65535,5300,50000";
+	char *resp3 = "%NCELLMEAS: 0,\"071D340C\",\"24201\",\"0821\",65535,5300,449,50,15,10891,655350";
```

Comment:
I would put one of the timing advance values in the tests to be higher than 32 bit limit so that we test decoding of int64_t more thoroughly.